src/sentry/conf/server.py

```diff
@@ -919,6 +919,8 @@ def create_partitioned_queues(name):
 # DSN for the frontend to use explicitly, which takes priority
 # over SENTRY_FRONTEND_PROJECT or SENTRY_PROJECT
 SENTRY_FRONTEND_DSN = None
+# DSN for tracking all client HTTP requests (which can be noisy) [experimental]
+SENTRY_FRONTEND_REQUESTS_DSN = None
```

Comment:
Do we have a getsentry pull to define this?

Reply:
Good call, added here, https://github.com/getsentry/getsentry/pull/3623